content/sensu-go/5.5/reference/filters.md

```diff
@@ -320,6 +320,47 @@ example      | {{< highlight shell >}} "annotations": {
   "playbook": "www.example.url"
 }{{< /highlight >}}
 
+### Event attributes
```

Comment:
As this is currently marked WIP, possible this is already being done downstream, but I think there needs to be more text here explaining exactly what these event attributes are. That they can be used inside a filter that a user writes and not available inside the context of a handler or mutator execution.

My first read of this, I thought it was talking directly about the built in filters, like `has_metrics` and `is_incident`, which led to confusion about the note and the examples listed.